Solver_interface/doc/Solver_interface/Concepts/SparseLinearAlgebraTraits_d.h

```diff
@@ -221,6 +221,10 @@ operator*(const NT& c, const Matrix& M);
 friend Matrix
 operator+(const Matrix& M0, const Matrix& M1);
 
+/// Assignment operator
```

Comment:
I though we only wanted to write that the concept ` Matrix`  refines ` Assignable`.

Reply:
I see, makes sense.
And the assignment operator alone is actually not sufficient as `Assignable` refines `CopyAssignable` and `CopyConstructible`.